fuzz/fuzz_targets/encode_decode.rs

```diff
@@ -2,20 +2,33 @@ extern crate bech32;
 
 use std::convert::TryFrom;
 
+use bech32::Hrp;
+
 fn do_test(data: &[u8]) {
     if data.len() < 1 {
+        println!("early return 0");
         return;
     }
 
-    let hrp_end = (data[0] as usize) + 1;
+    // Get a random valid length hrp 1-83 bytes.
+    let hrp_end = ((data[0] as usize) % 82) + 1;
```

Comment:
In 599ce6d089904386ac6a34d77a6b3b1fe222393f:

This actually only gives numbers in the range 1-82.

Reply:
ugh, code comments for the win!